kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -568,7 +568,21 @@ impl<'tcx> KaniAttributes<'tcx> {
 
     fn handle_stub_verified(&self, harness: &mut HarnessAttributes) {
         let dcx = self.tcx.dcx();
+        let mut seen = HashSet::new();
         for (name, def_id, span) in self.interpret_stub_verified_attribute() {
+            if seen.contains(&name) {
+                dcx.struct_span_err(
+                    span,
+                    format!("Multiple occurrences of `stub_verified({})`.", name),
+                )
+                .with_span_note(
+                    self.tcx.def_span(def_id),
+                    format!("Use a single `stub_verified({})` annotation.", name),
+                )
+                .emit();
+            } else {
```

Comment:
Can this be a warning instead? They will be redundant, but they can be safely ignored.

Reply:
Changed it to a warning, and moved the checks earlier in the process.